web/src/components/countryhistorypricesgraph.jsx

```diff
@@ -29,8 +30,8 @@ const prepareGraphData = (historyData) => {
   }));
 
   const layerKeys = ['price'];
-  const layerStroke = () => 'darkgray';
-  const layerFill = () => '#616161';
+  const layerStroke = undefined;
```

Comment:
It seems odd to have a const undefined, does everything break if you just gently remove it?